src/MemoryRouter.test.tsx

```diff
@@ -3,15 +3,16 @@ import { MemoryRouter } from "./MemoryRouter";
 describe("MemoryRouter", () => {
   const memoryRouter = new MemoryRouter();
   function currentRoute() {
-    const { pathname, query, asPath } = memoryRouter;
+    const { pathname, query, asPath, locale } = memoryRouter;
 
-    return { pathname, query, asPath };
+    return { pathname, query, asPath, locale };
```

Comment:
IIRC, the purpose of this `currentRoute` was so the inline snapshots were smaller.
However, since the scope of MemoryRouter is increasing, maybe we should replace `toMatchInlineSnapshot` with `toMatchObject`?  So we can "pick" what's appropriate for each test?

Reply:
make sense, working on the update

Reply:
Thanks, looks great!